src/QuickBench.hs

```diff
@@ -198,17 +200,31 @@ replaceExecutable exe cmd = (unwords (exe:args), exe, args) where args = drop 1
 time :: Opts -> String -> [String] -> IO Float
 time opts exe args = do
   t1 <- getCurrentTime
-  (c, o, e) <- readProcessWithExitCode' exe args ""
+  maybeOutput <- if (moreVerbose opts)
+    then Just <$> readCreateProcess (proc exe args) ""
+    else const Nothing <$> callProcessIgnoreOutput exe args
   t2 <- getCurrentTime
-  when (not $ null o) $ outvv opts $ (if verbose opts then "\n" else "") ++ o
-  unless (c == ExitSuccess) $ out opts $ " (error: " ++ clean e ++ ") "
```

Comment:
Instead of capturing `stderr`, I now let the child process inherit `stderr` from the parent process (the default). So any error messages written to `stderr` by say hledger will be verbatim written to the `stderr` of quickbench.

Reply:
The other change is that now, a failure of the childprocess will immediately result in failure of the parent process. I consider this an improvement, because before errors from the child process could get obscured by the normal output table. But perhaps you want to keep the original behavior @simonmichael ?

Reply:
Hmm. Should quickbench require that all tested commands succeed ? I think it's better to continue if some fail, because sometimes you're benchmarking many things and partial information is better than none.

I think it would be an improvement if it could show the failed status (or some of the error output, if it's not too disruptive of layout) in the results table though ?